app/views/products/_rating_view.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="pro-dec-rating">
+  <% while avg_rate > 0 %>
+    <% if avg_rate > 0.5 %>
+      <i class="ion-android-star theme-star"></i>
+    <% else %>
+      <i class="ion-android-star-half theme-star"></i>
+    <% end %>
+    <% avg_rate -= 1 %>
+  <% end %>
```

Comment:
avg_rate.to_i.times do In 1 sao end
((avg_rate % 1) > 0.5 ? (1 : 0)).times do In 0.5 sao end